src/class-tools/README.md

```diff
@@ -23,6 +33,7 @@ There is also the option to use `apply-parent-classes` or `data-apply-parent-cla
 ## Usage
 
 ```html
+<!-- The following DOM has classess swapped in and out as scheduled -->
```

Comment:
Seems you have a typo with an extra `s` here

Reply:
Done